### ISSUE TITLE
Not compatible with current Eureka Master (4.0.1)

### ISSUE THREAD

**Issue by dhmspector** [EurekaCommunity/RichTextRow#2]
Where I try to use this custom row I get an error from Cocoapods:

```
 RichTextRow (~> 1.0) was resolved to 1.0.0, which depends on   Eureka (= 3.0)
```

**Comment by fabfelici** [comment]
It should work now

**Comment by jaymathew** [comment]
This has not been fixed, I have been working with eureka 4.1.1 but when I added pod 'RichTextRow', '~> 1.0' Eureka got downgraded to 3.0. 

Am I missing something?

**Comment by fabfelici** [comment]
Hi, can you try using just `pod 'RichTextRow'`?

**Comment by adrianprause** [comment]
I am having the same issue.
RichTextRow (~> 1.0) was resolved to 1.0.0, which depends on
      Eureka (= 3.0)

**Comment by fabfelici** [comment]
Guys I created a brand new project with:
Podfile:
```
use_frameworks!

target 'Test' do
    pod 'RichTextRow', '~> 1.0'
end
```
and did a `pod install`:
```
Analyzing dependencies
Downloading dependencies
Installing Eureka (4.3.1)
Installing RichTextRow (1.0.3)
Generating Pods project
Integrating client project
```
Could you try from a brand new project as well?
